<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogGridappearance</class>
 <widget class="QDialog" name="dialogGridappearance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Appearance</string>
  </property>
  <widget class="QListWidget" name="BoxY1">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>65</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="BoxY1A">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>115</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="BoxY1B">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>165</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="BoxX">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>210</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="BoxBack">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>257</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="BoxBounds">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>257</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="BoxGrid">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>257</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="BoxY2A">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>115</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="BoxY2">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>65</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="BoxY2B">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>165</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="BoxText">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>257</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>52</y>
     <width>265</width>
     <height>155</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Use">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Use*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save as Preset*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>85</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>135</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y1A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>185</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y1B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Freq</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>277</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Background</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>277</y>
     <width>58</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Boundary</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>277</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>277</y>
     <width>26</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>185</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y2B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>135</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y2A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>85</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>5</y>
     <width>250</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Click on a color box to change it.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>20</y>
     <width>350</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>You may save this as a Custom Appearance.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>365</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>*Must save preferences file to make changes permanent.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
